Rockerfile.mesa

```diff
@@ -29,31 +31,73 @@ FROM {{ $image }}:llvm-{{ .LLVM }}
 
 LABEL maintainer "Juan A. Suarez Romero <jasuarez@igalia.com>"
 
-RUN apt-get update                                   \
-  && apt-get -y --no-install-recommends install git  \
-     {{ if eq .BUILD "windows" }}mingw-w64{{ end }}  \
+RUN apt-get update                                      \
+  && apt-get -y --no-install-recommends install git     \
+     {{ if eq .BUILD "windows" }}mingw-w64{{ end }}     \
+     {{ if eq .BUILD "android" }}                       \
+       wget openjdk-8-jdk python sudo make gcc gettext  \
+       python-mako bison libc6-dev xz-utils             \
+     {{ end }}                                          \
   && rm -fr /var/lib/apt/lists/*
 
+{{ if eq .BUILD "android" }}
+RUN adduser --gecos "" local && passwd -d local && adduser local sudo
+{{ end }}
 USER local
 
+WORKDIR /home/local
+
 {{ if .Env.MAKEFLAGS }}
 ENV MAKEFLAGS={{ .Env.MAKEFLAGS }}
 ENV SCONSFLAGS={{ .Env.MAKEFLAGS }}
 {{ end }}
 
+MOUNT {{ $ccachedir }}:/home/local/.ccache:Z
+
+{{ if eq .BUILD "android" }}
+RUN wget https://storage.googleapis.com/git-repo-downloads/repo \
+  && chmod u+x repo                                             \
+  && mkdir -p /home/local/.bin                                  \
+  && mv repo /home/local/.bin
+
+ENV PATH=/usr/lib/ccache:/home/local/.bin/:$PATH
+ENV USE_CCACHE=1
```

Comment:
Thinking more about it, I think I'll leave as it is. The thing is that I want to ensure that ccache is always the first in the PATH. If I set it at the beginning and later set local/.bin or llvm-bin (depending if it is Android or no) I don't know how to put it after the ccache path.

I'll push as it is, and if later we find a way to fix it, we will do.